### PR TITLE
Fixed #6266 -- Added missing migration operation

### LINKED ISSUE #6266: Missing migration for PageUserGroup  on django 1.10 & 1.11
Reported by @steffenmllr
Change introduced in bff63d55d1a871cd764f4061767c955b10882cce

### PR DESCRIPTION
**Pull request by czpython**
Fixed #6266

### CODE REVIEW

**Comment by timgraham** [comment]
To fix the test failures on Django < 1.9, I think the migration operation should be inserted only for Django 1.10 and later. By the way, the `exit_code`option is deprecated (and removed in Django 2.0) in favor of `check_changes` (https://github.com/django/django/commit/3c7d2ee881b13c7fdd0aa90b0aba0430310786de) but if you plan to follow the recommendation to drop support for Django < 1.11 before adding Django 2.0 support, then I'd modify the test at that time.

**Comment by czpython** [comment]
Thanks @timgraham.
I found it a bit confusing that the old exit code for no changes was 1 and now is 0.
We will support up-to django 1.11 on 3.5 and 1.11 / 2.0 on 3.6

File: cms/migrations/0018_pagenode.py [hunk 1]
```diff
@@ -165,3 +166,13 @@ class Migration(migrations.Migration):
             name='path',
         ),
     ]
+
+if (1, 10) <= django.VERSION < (2, 0):
```
Comment (timgraham) [inline cms/migrations/0018_pagenode.py]: I think `< (2, 0)` should be omitted.

File: cms/tests/test_migrations.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+            'interactive': False,
+            'dry_run': True,
+            'stdout': output,
+        }
+
+        if DJANGO_1_9:
+            # this was deprecated on Django 1.10
+            options['exit_code'] = True
+        else:
+            options['check_changes'] = True
```
Comment (timgraham) [inline cms/tests/test_migrations.py]: You could put this in the original dict and delete the key in the DJANGO_1_9 block so you can just remove that when dropping support for old versions.

**Comment by czpython** [comment]
Thanks. Changes applied.

File: cms/tests/test_migrations.py [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+        output = StringIO()
+        options = {
+            'interactive': False,
+            'dry_run': True,
+            'stdout': output,
+            'check_changes': True,
+        }
+
+        if DJANGO_1_9:
+            # this option was deprecated on Django 1.10
```
Comment (timgraham) [inline cms/tests/test_migrations.py]: on -> in

**Pull request merged**